theano/sandbox/cuda/tests/test_blas.py

```diff
@@ -26,12 +27,12 @@
 
 
 if theano.config.mode == 'FAST_COMPILE':
-    mode_with_gpu = theano.compile.mode.get_mode('FAST_RUN').including('gpu')
+    mode_with_gpu = theano.compile.mode.get_mode('FAST_RUN').including('gpu').excluding('cudnn')
     mode_without_gpu = theano.compile.mode.get_mode(
-            'FAST_RUN').excluding('gpu')
+            'FAST_RUN').excluding('gpu').excluding('cudnn')
 else:
-    mode_with_gpu = theano.compile.mode.get_default_mode().including('gpu')
-    mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpu')
+    mode_with_gpu = theano.compile.mode.get_default_mode().including('gpu').excluding('cudnn')
+    mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpu').excluding('cudnn')
```

Comment:
The current tests are passing without this excluding. So can you remove them? It was probably due to a temp problem with the commit at the base of this PR. Also rebase, I'll run the test on the rabased version to make sure all is good.